lib/database.rb

```diff
@@ -0,0 +1,24 @@
+require 'pg'
+require 'date'
+require 'time'
+class Database
+  def self.connect
+    if ENV['ENVIRONMENT'] == 'test'
+      connection = PG.connect(dbname: 'chitter_test')
+    else
+      connection = PG.connect(dbname: 'chitter')
+    end
+    return connection
+  end
+
+  def self.get_messages
+    connection = Database.connect
+    result = connection.exec("SELECT * FROM peeps ORDER BY date, time;")
```

Comment:
Would this be `ORDER BY date, time DESC` to have the most recent messages appear first?